docker-compose/realm-config/jhipster-realm.json

```diff
@@ -21,7 +21,7 @@
   "actionTokenGeneratedByAdminLifespan": 43200,
   "actionTokenGeneratedByUserLifespan": 300,
   "enabled": true,
-  "sslRequired": "none",
+  "sslRequired": "external",
```

Comment:
Wenn wir das hier machen, können wir uns in unserem deployten Live-Environment auf dem vServer nicht mehr bei Keycloak einloggen, weil das dann HTTPS verlangt. Das können wir aber leider nicht einfach konfigurieren bzw. das wäre mit erheblichem Aufwand verbunden.